### ISSUE TITLE
only one your image detect 

### ISSUE THREAD

**Issue by DGP13** [pukeanddie/aruco-arkit-localizer#4]
Hello

opencv2.framework issue is solve can you tell me why only one image detect in camera. Can you describe me how to work.

Thanks

**Comment by andyDW88** [comment]
Hi all, 
  i am not exactly sure but i figured out it might be the case he set marker NO.23 as the marker to be detected somewhere in the opencv wrapper. although anybody found the problem that when you move the marker the blue 3d model doesn't follow the marker like other aruco projects show on youtube?

**Comment by natwales** [comment]
Yes, that's correct it's set up to only detect one marker right now. You can change which one or add more easily - it should be commented in the code. I build this demo to be as simple an example as possible of a specific technique - to establish a real-world position for ARKit using the marker pose from a single corresponding frame. I had tried briefly to track the marker over multiple frames and ran into some performance issues. I'm sure it's a solvable problem, but for my use case was unnecessary. 

Also, it's worth looking into Vuforia which now finally supports extended tracking through ARKit. You'll find that that is also able to track the marker concurrently with arkit.

**Comment by andyDW88** [comment]
Hi Nat, 
  Sorry if i got the name wrong. Many thanks for the swift reply :) If you are interested in putting on some smooth tracking feature on the demo here is some of my work on the code: I tried implementing the renderer with the update at time parameter. Just to make it update the node's position every frame. Sadly it didn't work. I tried passing in the sceneView's current frame. The code would compile but it crashes every time although I really can't figure out why
